backend/amd64/selection.ml

```diff
@@ -346,6 +346,19 @@ method! select_operation op args dbg =
       Ispecific Izextend32, [arg]
     | _ -> super#select_operation op args dbg
     end
+  | Ccsel _ ->
+     begin match args with
+     | [cond; ifso; ifnot] ->
+       let (cond, earg) = self#select_condition cond in
+       (match cond with
+        | Ifloattest (w,CFeq) ->
+          (* CFeq cannot be represented as cmov without a jump.
```

Comment:
I think this should mention that `Icsel` with `CFneq` will do the move on either not-eq or unordered.

Reply:
I updated the comment.